### DISCUSSION TITLE
Solid particle drag reduction

### DISCUSSION THREAD

**Discussion by r-broek** [firemodels/fds#13103] [Q&A] [answered]
Hi,

Are any drag reduction models applied to (moving) solid Lagrangian particles, or are they only used for droplets? I am confused as the User's Guide lists it under Liquid Droplets, yet the Technical Reference Guide discusses a case that features solid spheres (9.1.2).

**Comment by mcgratta** [comment, marked as answer]
The `WAKE_REDUCTION` formulae of Section 9.1.2 are only applied to liquid droplets.

**Reply by r-broek** [reply]
Thank you.
